backend/apps/incidences/serializers.py

```
@@ -12,11 +12,29 @@ class IncidenceSerializer(serializers.ModelSerializer):
+    )
+
     class Meta:
         model = Incidence
         fields = [
             'id', 'title', 'description', 'location_type', 'room_number', 
-            'status', 'priority', 'updates', 'admin_notes', 'created_at', 'is_mine'
+            'status', 'priority', 'updates', 'admin_notes', 'created_at', 'is_mine',
+            'assigned_staff_name', 
+            'assigned_staff_job',
+            'assigned_external_name'
         ]
```

Ruff (0.15.5)

[warning] Mutable default value for class attribute

(RUF012)